src/jit/lsraarmarch.cpp

```diff
@@ -783,23 +780,33 @@ void Lowering::TreeNodeInfoInitPutArgSplit(GenTreePutArgSplit* argNode, TreeNode
 
     if (putArgChild->OperGet() == GT_FIELD_LIST)
     {
-        NYI_ARM("LSRA: Oper for split struct argument is GT_FIELD_LIST");
+        // Generated code:
+        // 1. Consume all of the items in the GT_FIELD_LIST (source)
+        // 2. Store to target slot and move to target registers (destination) from source
+        //
+        argNode->gtLsraInfo.srcCount = argInfo->numRegs + argInfo->numSlots;
```

Comment:
I implemented to change to GT_FIELD_LIST if the size of struct is less than 16 byte (or 32 byte if the struct is HFA) on global morphing phase. So there may not have register issue now.